context-color

```diff
@@ -98,10 +98,9 @@ __context_color_hash() {
     if [ "$CC_METHOD" = "md5sum" ]
     then
         hash=$(eval "$CC_CONTEXT" | md5sum | head -c 6 | tr '[:lower:]' '[:upper:]')
-        echo "ibase=16; $hash" | bc
+        echo "$((16#$hash))"
     else
         hash=$(eval "$CC_CONTEXT" | sum | cut -d' ' -f1)
```

Comment:
I think you forgot a line here!

```suggestion
        hash=$(eval "$CC_CONTEXT" | sum | cut -d' ' -f1)
        echo "$((10#$hash))"
```

Reply:
The reasons behind this are that 1/ we need to output something, and 2/ it must not start with a leading 0, which can happen with the result of sum.

Reply:
Thanks, didn't think about those corner cases :)